pkg/server/event_handler_test.go

```diff
@@ -89,53 +90,79 @@ func TestEventHandler_HttpHandler(t *testing.T) {
 	ctx := context.Background()
 
 	cases := []struct {
-		name               string
-		pubsubMessageBytes []byte
-		gcsObjectBytes     []byte
-		wantStatusCode     int
-		wantRespBodySubstr string
+		name                   string
+		pubsubMessage          *PubSubMessage
+		PubSubMessageBytes     []byte
+		gcsObjectBytes         []byte
+		gcsObjectMetadataBytes []byte
```

Comment:
Do we need this if it's embedded in the pubsub message data?

Reply:
Removed, as it's no longer needed.